rero_ils/query.py

```diff
@@ -50,6 +50,40 @@ def inner(values):
     return inner
 
 
+def and_term_filter_fiction(field):
+    """Create a term filter for subject (fiction).
+
+    :param field: Field name.
+    :return: Function that returns a boolean AND query between term values.
+    """
+    def inner(values):
+        must = [Q('term', **{field: value}) for value in values]
+        must.append(Q(
+            "terms",
+            **{'genreForm.identifiedBy.value': [
+                'A027757308', 'A021097366']}))
```

Comment:
Why hardcoded the values ?
Make a comment for values.
TODO: To transform this into the config ?